src/lib/config.ts

```diff
@@ -99,4 +108,19 @@ export default class Config implements ConfigInterface {
 		await this.database.setStringConfig(config, value);
 		await this.updateStringConfig();
 	}
+
+	async deleteRole(userid : string, role : Role) : Promise<void> {
+		await this.database.deleteRole(userid, role);
+		await this.updateRoles();
+	}
+
+	async deleteConfig(config : string) : Promise<void> {
+		await this.database.deleteConfig(config);
```

Comment:
What happens if the specified `config` does not exist?

Reply:
That's why we're doing checks for that now before we call the function